.github/workflows/linux.yml

```diff
@@ -0,0 +1,186 @@
+name: Linux
+
+on:
+  # allows us to run workflows manually
+  workflow_dispatch:
+  pull_request:
+    branches:
+      - develop
+      - master
+  push:
+    branches:
+      - develop
+      - master
+
+env:
+  GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
+jobs:
+  main:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        include:
+          - cubit: 17_1_0
+            os: ubuntu-18.04
+            URL: LINUX_CUBIT_17_1_0
+            URL_SDK: LINUX_CUBIT_SDK_17_1_0
+            
+          - cubit: 2020_2
+            os: ubuntu-20.04
```

Comment:
Is there a particular reason that different Ubuntu versions are used for these?

Reply:
Cubit 2020 is not compatible with package versions shipped with 18.04.

Reply:
So if anything but `ubuntu:20.04` is passed with `2020.2` should the script error or print a warning?

Reply:
I guess more recent version should be fine as well...
 And this isn not necessary, if one install the required deps....

anyway if the requirement are not met `apt-get` will through an error when trying to install Cubit.

I would keep that as is specially as the targeted audience of those scripts should have some experience with those tools (apt-get, docker etc...)

Reply:
Given the target audience for these scripts, I think that's fair.